raiden/tests/integration/network/proxies/test_service_registry.py

```diff
@@ -38,12 +38,12 @@ def test_service_registry_random_pfs(
 
     # Test that get_service_address by index works
     for idx, address in enumerate(addresses):
-        c1_service_proxy.get_service_address('latest', idx) == address
+        assert c1_service_proxy.get_service_address('latest', idx) == address
```

Comment:
good catch!

Reply:
I wonder if there is a tool to detect this error. Should be easy enough, a boolean comparison that stand on its own as a statement and not used anywhere should be a bug.